manifests/server/resources.pp

```diff
@@ -1185,6 +1187,13 @@
     }
   }
 
+  if $manage_cilium {
+    if $manage_kube_proxy { fail('please disable manage_kube_proxy') }
```

Comment:
Nice warning! Another option would be to check if multiple options are set, and switch these in a central place like:
`$foo = !$bar`. It's the same use case for the container engine.

Reply:
if nobody else wants to change this i like to have it this way. i dont like these `if $something and !$something and $smthing and and or not and or not or and or` constructs. 😄